src/amo/actions/landing.js

```diff
@@ -1,31 +1,38 @@
 import {
   LANDING_GET,
   LANDING_LOADED,
-  LANDING_FAILED,
 } from 'core/constants';
 
 
-export function getLanding({ addonType }) {
+export function getLanding({ addonType, errorHandlerId }) {
   if (!addonType) {
     throw new Error('addonType must be set');
   }
+  if (!errorHandlerId) {
+    throw new Error('errorHandlerId must be set');
+  }
 
   return {
     type: LANDING_GET,
-    payload: { addonType },
+    payload: { addonType, errorHandlerId },
   };
 }
 
 export function loadLanding({ addonType, featured, highlyRated, popular }) {
+  if (!addonType) {
+    throw new Error('addonType cannot be empty');
+  }
+  if (!featured) {
+    throw new Error('featured cannot be empty');
```

Comment:
Do you think adding `Addons` to the end of each variable would help with readability? `featured cannot be empty` is a bit ambiguous to me. 🤷‍♀️